tests/010-test-use-devs-to-create-docker-root-volume.sh

```diff
@@ -19,6 +19,7 @@ test_docker_root_volume() {
 DEVS="$devs"
 VG=$vg_name
 DOCKER_ROOT_VOLUME=yes
+DOCKER_ROOT_VOLUME_SIZE=40%FREE
```

Comment:
Why are we setting `DOCKER_ROOT_VOLUME_SIZE=40%FREE`? I believe this should be picked by default.